pages/jdl.md

```diff
@@ -280,7 +280,7 @@ Latest version introduces exclusions (which is quite a powerful option when sett
     paginate C with pagination
 
 
-With JHipster, you can also tell whether you don't want any client code, or server code. Even if you want to add a suffix to Angular-related files, you can do that in JHipster.
+With JHipster, you can also tell whether you don't want any client code, or server code. Even if you want to add a suffix to Angular-related files, you can do that in JHipster. [Filtering](https://www.jhipster.tech/entities-filtering/) options can be activated on a per entity basis: filter `<entity name>` or for all entities: filter `*`.
```

Comment:
Small nitpick but I would put the code tags around the full JDL, so:

`filter *` instead of "filter `*`" 
`filter <entity name>` instead of "filter `<entity name>`"

Reply:
Ok, i did the changes you suggest and commit. It that it or should I do something else? Thanks Jon.